packages/redux/src/recommendations/__tests__/reducer.test.ts

```diff
@@ -48,19 +50,21 @@ describe('recommendations reducer', () => {
     });
 
     it('should handle other actions by returning the previous state', () => {
-      const state = {
+      const state: State = {
         error: {
           [mockRecommendationsStrategy]: error,
```

Comment:
Typescript is complaining that `error` is a string and not an `Error` instance. Can you change it to an Error, please?

Reply:
Source updated, can you check again.